homeassistant/components/wiz/translations/he.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
These will disappear one rebase. Its just out of sync with `dev` because of the dep chain